code/datums/gamemodes/conspiracy.dm

```diff
@@ -5,6 +5,7 @@
 	latejoin_only_if_all_antags_dead = 1 // No hunters until the conspiracy is dead, thanks
 
 	var/maxConspirators = 6
+	var/agent_radiofreq = 1401
```

Comment:
why not use the define?

Reply:
Actually now that I think about it, I don't even use my define anywhere and I don't really need it. Removed!